src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubectlBuildWrapper.java

```diff
@@ -68,13 +68,15 @@
     private final String serverUrl;
     private final String credentialsId;
     private final String caCertificate;
+    private final String https_proxy;
```

Comment:
Makes me feel uneasy. I'd prefer something more generic like providing a list of environment variables overrides.